Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -521,11 +521,10 @@ class BrowserViewController: UIViewController {
         super.viewDidAppear(animated)
         log.debug("BVC done.")
 
-        let appVersion = AppInfo.appVersion
-        if profile.prefs.stringForKey("latestAppVersion") != appVersion && DeviceInfo.hasConnectivity() {
+        if profile.prefs.stringForKey(LatestAppVersionProfileKey) != AppInfo.appVersion && DeviceInfo.hasConnectivity() {
```

Comment:
The logic here is not correct. It should be: *If `LatestAppVersionProfileKey` is not nil and not the same as the current app version, then we show the What's New page.*